lessons/Early-Math/equivalence/langs/en-us/index.adoc

```diff
@@ -240,8 +289,75 @@ There are numerous benefits to inviting students to search out similarities and
 If your students are getting stuck, start a conversation! What do students notice, at first glance? What makes the Circles of Evaluation alike, and what makes them different? Have students share their reasoning to create opportunities for peer learning.
 }
 
+@slidebreak
 
 === Synthesize
 
 - What strategies did you use to determine whether or not Circles of Evaluation were equivalent?
 - Did you find that some strategies were more efficient than others? Why?
+
+@slidebreak
+
+== Programming Exploration: Are They Equivalent? @duration{20 minutes}
+
+=== Overview
+
+Extending concepts explored earlier in the lesson, students consider what makes one line of code equivalent to another.
+
+=== Launch
+
+@slidebreak
+
+In the first section of this lesson, you explored different ways of creating Circles of Evaluation that are equivalent to 16. Computation was a valuable tool for determining equivalence... but considering _structure_ helped us recognize equivalence, too!
+
+What if a programmer wants to determine whether two lines of code will produce the same result? They have two options. They can test the code and see what happens. More experienced programmers, however, study the structure of the code. They think about the functions being used... and then come to a conclusion. These programmers are more strategic and efficient when they write their own code.
+
+@slidebreak
+
+=== Investigate
+
+
+When we learned about _equivalence_, we considered varying @vocab{arithmetic expressions} and thought about whether they would simplify to the same value or not.
+
+When we talk about images - like the ones we can produce with code - we need some different terminology. Various shapes can have equivalent _areas_ or equivalent _side lengths_ because _area_ and _length_ are represented by numbers.
+
+@slidebreak
+
+To describe two shapes that are exactly identical, we need a different word: *congruent*. Two images are congruent if they are perfectly the same. And _congruent_ images are produced by _equivalent_ lines of code!
```

Comment:
Congruent triangles are _not_ exactly identical. I think the word you're looking for is "identical". Equivalent expressions evaluate to identical values. Equivalent image-producing code evaluate to identical images.

Reply:
you are right. Thanks.